test/src/main/java/org/axonframework/test/aggregate/AggregateTestFixture.java

```diff
@@ -311,16 +318,13 @@ public AggregateTestFixture(Class<T> aggregateType) {
 
     @Override
     public TestExecutor<T> givenState(Supplier<T> aggregate) {
+        if(this.repository == null) {
```

Comment:
Ideally, we'd throw an `AxonConfigurationException` if `useStateStorage()` hasn't been invoked.
However, this would be a breaking change for other users when they increment to a newer Axon version, so we cannot do that.

Hence, for now, I think what you've done here is fine. I'll add it in our internal documentation that we adjust this portion to require the usage of the `useStateStorage()` in Axon 5.x (once we start construction on this).